### DISCUSSION TITLE
Performing CreateMultipartUploadInput, UploadPart, and CompleteMultipartUpload with aws-sdk-go-v2

### DISCUSSION THREAD

**Discussion by wyim-ns** [aws/aws-sdk-go-v2#2581] [Q&A]
Our existing code is using aws-sdk-go V1.  We don't get the entire large file at once and we rely on CreateMultipartUploadInput, UploadPart, and CompleteMultipartUpload on our own.  I can't any of these calls in aws-sdk-go-v2.  We can't use the new v2's s3/manager NewUploader.  We don't have the entire large file on any ONE given server.  We rely on the uploadId and PartNumber for resembling the entire large file.

Is there any documentation or sample code to perform the same "Multipart Upload" with aws-sdk-go-v2?

**Comment by lucix-aws** [comment]
See https://github.com/aws/aws-sdk-go-v2/issues/1469#issuecomment-1960349975

**Comment by wyim-ns** [comment]
Hi @lucix-aws , thank you for the response. But isn't python boto3 still supports “create_multipart_upload()" and "upload_part()"?
